packages/internal/src/__tests__/graphqlCodeGen.test.ts

```diff
@@ -1,6 +1,8 @@
 import fs from 'fs'
 import path from 'path'
 
+import { getConfig } from '@testing-library/react'
```

Comment:
Is this correct? Think import-path should be project-config!

Reply:
It was mocked in the tests, so never used! Changin